.github/workflows/start-script.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# TODO: replace with master branch after merge
+curl -O- https://github.com/duckduckgo/sync-config/blob/lkruger/sync1/test/setup.sh
```

Comment:
`sync_config` is a private repo, at least for now. A curl with no authentication will yield a 404 response.